src/docs/metrics/social-data/index.md

```diff
@@ -50,6 +56,8 @@ For each of the data sources we have a curated list of accounts/sub-sources from
 
   **History**: The historical data starts at `2016-01-01`.
 
+  **Availability**: Data is currently only included in parts of Sanbase ([https://app.santiment.net/labs/trends/explore/bullrun](https://app.santiment.net/labs/trends/explore/bullrun)) and Sandata.
```

Comment:
Reddit is also present in the new Sangraphs, so I guess it's present pretty much everywhere

Reply:
Is it included in the Social Volume scores on Sanbase by now? @ivanklimuk

Reply:
@balresch It's tricky because we do have reddit on Sanbase in the "Labs" section, but we don't have it on the project overview page yet included - only a few users noticed it, but still

Reply:
```suggestion
  **Availability**: Data is currently only included in parts of Sanbase ([https://app.santiment.net/labs/trends/explore/bullrun](https://app.santiment.net/labs/trends/explore/bullrun)) and Sandata, as well as our experimental platform [https://graphs.santiment.net/social](https://graphs.santiment.net/social).
```

Reply:
So would the above be correct at this point in time? @ivanklimuk

Reply:
@ivanklimuk Can I commit the above changes?

Reply:
I think yes, thank you!